packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -133,7 +133,7 @@ export const Components = {
       title: (title: string) => `Drawer title ${title}`,
       expand: 'Drawer expand',
       contract: 'Drawer contract',
-      close: 'Drawer close',
+      close: 'data-testid Drawer close',
```

Comment:
Some tests are failing and I figured out it is because this e2e selector is also used in the `Drawer` component. So if we want those tests to pass we need to also change that `Button` to look like this one or we need to create a separate selector for the `NewsContainer` drawer. If we could reuse the same it would be great though 👍

Reply:
@JoaoSilvaGrafana, that works. I think it's best if we use the same selector for both as it was before; that means I should update the `Drawer` component to be something like this:
```diff
- <Button
-    icon="times"
     variant="secondary"
     fill="text"
     onClick={onClose}
-    aria-label={selectors.components.Drawer.General.close}
   />
+ <IconButton
+    name="times"
     variant="secondary"
     fill="text"
     onClick={onClose}
+    data-testid={selectors.components.Drawer.General.close}
+    tooltip={t(`drawer.close`, 'Close Drawer')}
+    />
```
Correct?

Reply:
Yes that seems perfect! Thanks!

Reply:
Hi @JoaoSilvaGrafana, changes made